docs/en-us/development/development-environment-setup.md

```diff
@@ -0,0 +1,148 @@
+```
+    spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+    spring.datasource.url=jdbc:mysql://localhost:3306/dolphinscheduler
+    spring.datasource.username=ds_user
+    spring.datasource.password=dolphinscheduler  
+```
+
+3. Modify the pom.xml under the root directory, change the scope of the mysql-connector-java to complie
+
+4. Refresh the dao module  
+try to run the main method inside org.apache.dolphinscheduler.dao.upgrade.shell.CreateDolphinScheduler to insert tables and data needed for the project.
+
+5. Modify the service module  
+try to change the zookeeper.quorum part of the zookeeper.properties file
+zookeeper.quorum=localhost:2181
+
+6. Modify the dolphinscheduler-ui module  
+try to change the .env file
+```
+    API_BASE = http://localhost:12345
+    DEV_HOST = localhost
+```
+
+#### Start the project
+
+1. Start zookeeper  
+./bin/zkServer.sh start
+
+2. Start MasterServer  
+try to run the main method inside org.apache.dolphinscheduler.server.master.MasterServer and VM Options is needed:
+```
+    -Dlogging.config=classpath:logback-master.xml -Ddruid.mysql.usePingMethod=false
+```
+3. Start WorkerServer  
+try to run the main method inside org.apache.dolphinscheduler.server.worker.WorkerServer and VM Options is needed:
+```
+    -Dlogging.config=classpath:logback-worker.xml -Ddruid.mysql.usePingMethod=false
+```
+4. Start ApiApplicationServer  
+try to run the main method inside org.apache.dolphinscheduler.api.ApiApplicationServer and VM Options is needed:
+```
+    -Dlogging.config=classpath:logback-api.xml -Dspring.profiles.active=api
+```
+5. We are not going to start the other modules. if they are required to be started, check script/dolphinscheduler-daemon.sh and set them the same VM Options.
+```
+    if [ "$command" = "api-server" ]; then
+      LOG_FILE="-Dlogging.config=classpath:logback-api.xml -Dspring.profiles.active=api"
+      CLASS=org.apache.dolphinscheduler.api.ApiApplicationServer
+    elif [ "$command" = "master-server" ]; then
+      LOG_FILE="-Dlogging.config=classpath:logback-master.xml -Ddruid.mysql.usePingMethod=false"
+      CLASS=org.apache.dolphinscheduler.server.master.MasterServer
+    elif [ "$command" = "worker-server" ]; then
+      LOG_FILE="-Dlogging.config=classpath:logback-worker.xml -Ddruid.mysql.usePingMethod=false"
+      CLASS=org.apache.dolphinscheduler.server.worker.WorkerServer
+    elif [ "$command" = "alert-server" ]; then
+      LOG_FILE="-Dlogback.configurationFile=conf/logback-alert.xml"
+      CLASS=org.apache.dolphinscheduler.alert.AlertServer
+    elif [ "$command" = "logger-server" ]; then
+      CLASS=org.apache.dolphinscheduler.server.log.LoggerServer
+    else
+      echo "Error: No command named \`$command' was found."
+      exit 1
+    fi
+```
+
+6. Start the front-end ui modules  
+cd dolphinscheduler-ui directory
+And run 
+npm run start
```

Comment:
cd dolphinscheduler-ui directory and run `npm run start`

Reply:
done changes